app/frontend/src/features/communities/constants.ts

```diff
@@ -11,6 +11,7 @@ export const communityTabBarLabels: Record<CommunityTab, string> = {
 };
 
 export const getByCreator = (name: string) => `By ${name}`;
+export const getAttendeesCount = (count: number) => `${count} attendees`;
```

Comment:
```suggestion
export const getAttendeesCount = (count: number) => count === 1 ? "1 attendee" : `${count} attendees`;
```
Proper i18n "Coming Soon™"